src/components/TeamMemberTasks/components/TaskCompletedModal.jsx

```diff
@@ -1,31 +1,79 @@
-import React from 'react';
 import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import React from 'react';
+import { useEffect } from 'react';
+import { ENDPOINTS } from 'utils/URL';
+import axios from 'axios';
 
 /**
  * Modal popup to delete the user profile
  */
-const DuplicateNamePopup = React.memo(props => {
+const TaskCompletedModal = React.memo(props => {
   const closePopup = e => {
-    props.onClose();
+    props.popupClose();
+  };
+
+  const loadUserTasks = async id => {
+    const userId = id;
```

Comment:
Just a small question, why do you create `userId` rather than directly use `id`? Thank you.